From: Marcel Holtmann <marcel@holtmann.org>
Date: Sun, 11 Feb 2018 21:03:04 +0100
Subject: build: Add extra dependency for internal ELL library

---
 Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 3fc6db9..43caed1 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -106,8 +106,10 @@ src_iwd_SOURCES = src/main.c linux/nl80211.h \
 
 src_iwd_LDADD = ell/libell-internal.la -ldl
 
+src_iwd_DEPENDENCIES = ell/libell-internal.la
+
 if SYSTEMD_SERVICE
-src_iwd_DEPENDENCIES = src/iwd.service
+src_iwd_DEPENDENCIES += src/iwd.service
 
 systemd_unitdir = @SYSTEMD_UNITDIR@
 systemd_unit_DATA = src/iwd.service
